src/MessageTrait.php

```diff
@@ -112,7 +112,7 @@ public function getHeaders()
      */
     public function hasHeader($header)
     {
-        return array_key_exists(strtolower($header), $this->headerNames);
+        return isset($this->headerNames[strtolower($header)]);
```

Comment:
I have added test for that - still working as expected (`testHeaderExistsIfWithNoValues`, `testHeaderWithNoValues`). Do you have any other test case to fall it?